docs/usage/project-operations.md

```diff
@@ -16,7 +16,7 @@ This section demonstrates how to use the standard Kubernetes tool for cluster op
 - You’re logged on to the Gardener Dashboard.
 - You’ve created a cluster and its status is operational.
 
-It's recommended that you get acquainted with the resources in the [Gardener API](https://gardener.cloud/api-reference/).
+It's recommended that you get acquainted with the resources in the [Gardener API](https://github.com/gardener/gardener/tree/master/docs/api-reference).
```

Comment:
```suggestion
It's recommended that you get acquainted with the resources in the [Gardener API](https://gardener.cloud/docs/gardener/api-reference/).
```

Reply:
Links should point to the github repository instead of the website, as pointed out [here](https://github.com/gardener/gardener/pull/4951#discussion_r744953376).